sources/osgShadow/ShadowMap.js

```diff
@@ -236,7 +236,7 @@ define( [
 
         // Overridable Visitor so that user can override the visitor to enable disable
         // in its own shadowmap implementation
-        this._removeNodesNeverCastingVisitor = new RemoveNodesNeverCastingVisitor( this._castsShadowTraversalMask );
+        this._removeNodesNeverCastingVisitor = settings.userCallbackNodeCaster || new RemoveNodesNeverCastingVisitor( this._castsShadowTraversalMask );
```

Comment:
It's better to use setter, it's more the way used in osg/js to change content. do you use settings.userCallbackNodeCaster somewhere in the code ?